tests/testthat/test-download-data-cube.r

```diff
@@ -0,0 +1,36 @@
+temp_path <- tempdir()
+
+test_that("File is downloaded",{
+  #Download May 2020 EQ09
+  temp_path <- tempdir()
+
+  download_abs_data_cube(cat_no = "6291.0.55.003",
```

Comment:
Hi @daviddiviny, the second test (line 19) is failing on Travis. I think it's because the temp dir gets flushed between tests. To fix this, you could either: 

1. Move line 7 (`download_abs_data_cube()`) so that it is not inside a `test_that()` call (ie. move to line 2) or 
2. Repeat the `download_abs_data_cube()` call within each `test_that()` call. 

I'd go with option 1.

Reply:
I tried both Options and it doesn't seem to be working?